features/step_definitions/autocomplete_steps.rb

```diff
@@ -34,154 +39,178 @@
 # Define all values to be entered here depending on the fieldname
 When /^I enter text in the "([^\"]+)" autocomplete field$/ do |fieldname|
   text = case fieldname
-    when "Fandoms"
-      "sup"
-    when "Additional Tags"
-      "alt"
-    when "Characters"
-      "ellen"
-    when "Relationships"
-      "stiel"
-    when "Your Tags"
-      "sup"
-    else
-      ""
-    end
-  step %{I fill in "#{fieldname}" with "#{text}"}
+         when "Fandoms"
+           "sup"
+         when "Additional Tags"
+           "alt"
+         when "Characters"
+           "ellen"
+         when "Relationships"
+           "cast"
```

Comment:
I changed "stiel" to "cast". We set up Redis to look for prefixes for each character in a ship ("c", "ca", "cas", etc), searching for a suffix doesn't work.